week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,12 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  return price * 1.3 * (1 + 1 / 100);
```

Comment:
Correct! Can you simplify `1 + 1 / 100` at all to reduce the number of calculations being done on this line?